src/components/enterprise-user-subsidy/OffersAlert.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
I think this should be its own standalone component that is rendered as a one of the children of `UserSubsidy`, like:

```
<EnterpriseBanner />
<UserSubsidy>
  <OffersAlert />
</UserSubsidy
```

Reply:
This is also not quite what we'd want since I would expect the `SubscriptionSubsidy` alerts to come after the `OffersAlert` in the UI. 

I think we should revert the PR causing the issue and figure out a path forward for the fix on Monday.

![image](https://user-images.githubusercontent.com/2828721/96308061-ac1b0c00-0fd0-11eb-8de4-d279f3a6a3eb.png)